pkg/apis/workflow/v1alpha1/workflow_template_types.go

```diff
@@ -16,6 +16,11 @@ type WorkflowTemplate struct {
 	Spec              WorkflowTemplateSpec `json:"spec" protobuf:"bytes,2,opt,name=spec"`
 }
 
+type WorkflowTemplateInterface interface {
+	GetTemplateByName(name string) *Template
+	GetTemplateScope() string
+}
+
```

Comment:
I think this should be on `workflow_types.go`

Reply:
yes, I will create common.go and move the interface. This is common for WFT and CWFT